cmd/record.go

```diff
@@ -56,7 +56,9 @@ func (t *Record) GetRecordConfig(path *string, proxyPort *uint32, appCmd *string
 	if len(*path) == 0 {
 		*path = confRecord.Path
 	}
-	filters = confRecord.Filters
+
+	filters = confRecord.Tests
```

Comment:
As the Tests type is changed is the flow changed according to it..?

Reply:
yes, tested as well (mentioned in description)